plugins/RnD/ColorMatching.py

```diff
@@ -0,0 +1,1167 @@
+1.4356626609467742,
+1.4283135748479714,
+1.4221174206706533,
+1.419088123941108,
+1.4151713096601313,
+1.4104479227564815,
+1.4067959767890836,
+1.4014983138828738,
+1.395993977947625,
+1.389172656234708,
+1.3843633987559076,
+1.3822994163135525,
+1.380050005073384,
+1.3751007814080232,
+1.3698199160412685,
+1.3640398945807293,
+1.3605436252334573,
+1.3581173312519927,
+1.3544017329241291,
+1.3511295028754058,
+1.3465049081285416,
+1.340691515616876,
+1.3356299458326444,
+1.3295617992959077,
+1.3242991054105424,
+1.3200322834911262,
+1.3140546070289025,
+1.3071294971525622,
+1.3015762174900993,
+1.2975397026719684,
+1.2934101921406413,
+1.2886980785981745,
+1.2815329778533298,
+1.2734617985706767,
+1.2672495792301905,
+1.262931337395203,
+1.259342189598978,
+1.2541017333242235,
+1.2498233752050452,
+1.244333501162457,
+1.2380405990846748,
+1.2336646582295137,
+1.2285778331609305,
+1.2236890494328747,
+1.2189851928081863,
+1.2148886487089845,
+1.2100926055932593,
+1.2061927872808689,
+1.2012160148978583,
+1.1966270196928963,
+1.1934182112927403,
+1.1894843871161496,
+1.18691129180388,
+1.1817266109898317,
+1.1771971151932985,
+1.171951828608877,
+1.1687693899168676,
+1.1653828661634056,
+1.1604564715871872,
+1.1548246709644936,
+1.149074358917443,
+1.1449900200144665,
+1.1411367431756856,
+1.137165339120491,
+1.1334915035920181,
+1.1295277881013972,
+1.1236953226007347,
+1.117965017127736,
+1.1139606116762628,
+1.1094983984052793,
+1.1057409663557298,
+1.1004019324718592,
+1.0955315423518355,
+1.0908357175119239,
+1.0867681742474617,
+1.083122871200466,
+1.0803253835179827,
+1.0758685220581483,
+1.072356266518207,
+1.068748191916973,
+1.0662567091794002,
+1.0617036629873295,
+1.0575073917563091,
+1.0530128051819863,
+1.0515231246620327,
+1.0476167496599287,
+1.0424254862532176,
+1.0370809071509972,
+1.0323910890637593,
+1.0275159450764026,
+1.0236921175914593,
+1.0186568605893738,
+1.0139075888384232,
+1.009764023356815,
+1.0047800783549203,
+1.0004232702016043,
+0.9971810499390729,
+0.9937981609657843,
+0.9904688959515542,
+0.9877438709970183,
+0.9844661017971743,
+0.9814705627505018,
+0.9790350032025117,
+0.9748246503023107,
+0.9699679481860232,
+0.9653797146884857,
+0.9610494990987312,
+0.9577422129174501,
+0.9550275898592997,
+0.9537206700881192,
+0.9497303904062503,
+0.9442702682104231,
+0.9412257015518758,
+0.938391189457947,
+0.9357202016619977,
+0.9326190350739556,
+0.9300196714424732,
+0.927444559208246,
+0.9252602575622887,
+0.9229621052048547,
+0.9171357616783294,
+0.9120024753827068,
+]
+
+COLOR_MODEL_ALIASES = {
+                        'CIE 1931 2 deg' : 'CIE 1931 2 Degree Standard Observer',
+                        'CIE 1931 10 deg' : 'CIE 1964 10 Degree Standard Observer',
+                        'CIE 2015 2 deg' : 'CIE 2015 2 Degree Standard Observer',
+                        'CIE 2015 10 deg' : 'CIE 2015 10 Degree Standard Observer'
+}
+
+class ColorMatching(EnlightenPluginBase):
+
+        def disconnect(self):
```

Comment:
I think you can delete this func, and just use the base class's.

Reply:
Done